Header header
float32 latitude
float32 longitude
float32 altitude
int64 utm_easting
int64 utm_northing
int64 fix_quality
string zone
